d2l-scroll-wrapper.js

```diff
@@ -213,7 +213,7 @@ $_documentContainer.innerHTML = `<dom-module id="d2l-scroll-wrapper">
 			<d2l-table-circle-button class="right action" icon="[[endIcon]]" on-tap="handleTapRight" tabindex="-1" aria-hidden="true" type="button"></d2l-table-circle-button>
 		</d2l-sticky-element>
 		<div id="wrapper" class="wrapper">
-			<div class="inner-wrapper" role$="[[_role]]"><slot></slot></div>
+			<div class="inner-wrapper"><slot></slot></div>
```

Comment:
Once manager dashboard's usage is gone, I should be able to remove this wrapper element entirely (and the style mixin).